utils/generate_tables.py

```diff
@@ -33,20 +33,43 @@ def get_data_size_and_hpss(hpss_path: str) -> Tuple[str, str]:
             hpss = ""
         return (data_size, hpss)
 
-def get_esgf(source_id: str, model_version: str, experiment: str, ensemble_num: str, cmip_only: str) -> str:
+def get_esgf(source_id: str, model_version: str, experiment: str, ensemble_num: str, link_type: str) -> str:
     if experiment and ensemble_num:
         # See https://github.com/E3SM-Project/CMIP6-Metadata/pull/9#issuecomment-1246086256 for the table of ensemble numbers
-        # remove v from model_version
-        esgf = f"`Native <https://esgf-node.llnl.gov/search/e3sm/?model_version={model_version[1:]}_0&experiment={experiment}&ensemble_member=ens{ensemble_num}>`_"
+        # Note that `[1:]`` removes `v` from `model_version`
+        # 
+        # Query parameters:
+        # v2:
+        # Native (model_version, experiment, ensemble_member)
+        # CMIP6 (source_id, experiment_id, variant_label)
+        # v1 on https://e3sm.org/data/get-e3sm-data/released-e3sm-data/v1-1-deg-data-cmip6/:
+        # Native (experiment, and optionally model_version)
+        # CMIP6 (activity_id, source_id, experiment_id)
+
+        # TODO: figure out if we're returning the correct things
```

Comment:
I've listed the numbers of items returned in the comments below, for e3sm.org vs the current implementation in this PR. As the table below reveals, the number of returned items don't always match up as expected.